deploy/olm-catalog/next/ember-csi-operator.vX.Y.Z.clusterserviceversion.yaml

```diff
@@ -0,0 +1,9203 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    containerImage: docker.io/embercsi/ember-csi-operator:latest
+    categories: "Storage"
+    capabilities: Basic Install
+    description: |-
+         Operator to deploy Ember-CSI, a multi-vendor CSI plugin driver supporting over 80 storage drivers in a single plugin to provide block and mount storage to container orchestration systems
+    certified: "false"
+    support: http://readthedocs.org/projects/ember-csi/
+    repository: https://github.com/embercsi/ember-csi-operator
+    createdAt: 2019-08-12:22:09:00
+    # BEGIN AUTO GENERATED EXAMPLES
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "ember-csi.io/v1alpha1",
+          "kind": "EmberCSI",
+          "metadata": {
+            "name": "example"
+          },
+          "spec": {
+            "config": {
+              "envVars": {
+                "X_CSI_PERSISTENCE_CONFIG": {
+                  "storage": "crd"
+                },
+                "X_CSI_DEBUG_MODE": "",
+                "X_CSI_DEFAULT_MOUNT_FS": "ext4",
+                "X_CSI_EMBER_CONFIG": {
+                  "grpc_workers": 30,
+                  "disable_logs": false,
+                  "plugin_name": "",
+                  "slow_operations": true,
+                  "disabled": "",
```

Comment:
This fails in the operator because it should be an empty list []. I'll make this more robust in the operator

Reply:
Done.